sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeHealthcareAsyncClient.java

```diff
@@ -196,12 +208,13 @@
             null);
     }
 
-    PollerFlux<TextAnalyticsOperationResult, Void> beginCancelAnalyzeHealthcare(UUID jobId, Context context) {
+    PollerFlux<TextAnalyticsOperationResult, Void> beginCancelAnalyzeHealthcare(String healthTaskId,
+        RecognizeHealthcareEntityOptions options, Context context) {
         try {
-            Objects.requireNonNull(jobId, "'jobId' is required and cannot be null.");
+            Objects.requireNonNull(healthTaskId, "'healthTaskId' is required and cannot be null.");
             return new PollerFlux<>(
-                DEFAULT_POLL_DURATION,
-                activationOperation(service.cancelHealthJobWithResponseAsync(jobId,
+                options == null ? DEFAULT_POLL_INTERVAL : options.getPollInterval(),
+                activationOperation(service.cancelHealthJobWithResponseAsync(UUID.fromString(healthTaskId),
```

Comment:
Is there a way to catch the potential exception that could be caused by the creation of UUID from string?

Reply:
we have try-catch statement around the code section.